lib/command_runner.dart

```diff
@@ -376,15 +396,15 @@ abstract class Command<T> {
 
   /// Throws a [UsageException] with [message].
   void usageException(String message) =>
-      throw new UsageException(message, _usageWithoutDescription);
+      throw new UsageException(_wrap(message), _usageWithoutDescription);
```

Comment:
I'm iffy about wrapping the message in the exception text. Exceptions are often caught programmatically. If you were to, say, log the message text to a file, you probably don't want it arbitrarily line-wrapped.

Is it possible to defer the wrapping until the exception is displayed instead of doing it eagerly here?

Reply:
OK, I'm convinced. Let's do it the way you suggest.